cli/mobilecoin/external_pb2.py

```diff
@@ -0,0 +1,1419 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Do we want to commit the generated code? Maybe that's the most expedient?

Reply:
For now it is, but as I mentioned in the PR description this is janky